include/dlfcn.h

```diff
@@ -309,6 +321,26 @@ int dlclose(FAR void *handle);
 
 FAR char *dlerror(void);
 
+/****************************************************************************
+ * Name: dladdr
+ *
+ * Description:
+ *   dladdr() provides information about the address of a symbol in a
+ *   dynamically loaded object.
+ *
+ * Input Parameters:
+ *   addr - The address of the symbol for which information is desired.
+ *   info - A pointer to a Dl_info structure that is filled in by dladdr().
+ *
+ * Returned Value:
+ *   On success, these functions return a nonzero value.
+ *
+ * Reference: OpenGroup.org
+ *
+ ****************************************************************************/
+
+int dladdr(const FAR void *addr, FAR Dl_info *info);
```

Comment:
DL_info_t?

Reply:
It's a little confuse the `Dl_info_t` is from POSIX, but `Dl_info` seems used more widely:
https://github.com/rust-lang/libc/blob/0e28c864c25d2e9b0ab082947445efccef213da4/src/unix/mod.rs#L1394

Reply:
then it's better to explain it in the above comment i guess.